platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -306,6 +313,80 @@ private void configureConduit(ClientConfiguration clientConfig) {
                 }
             }
         }
+
+        TLSClientParameters tlsParams = httpConduit.getTlsClientParameters();
+        if (tlsParams == null) {
+            tlsParams = new TLSClientParameters();
+        }
+
+        tlsParams.setDisableCNCheck(disableCnCheck);
+
+        tlsParams.setUseHttpsURLConnectionDefaultHostnameVerifier(true);
+        tlsParams.setUseHttpsURLConnectionDefaultSslSocketFactory(true);
+        tlsParams.setCipherSuites(Arrays.asList(System.getProperty("https.cipherSuites")
```

Comment:
Me too! I'm fairly certain that it did at one point. I'm like 99% sure all of this worked just fine without us setting any of this stuff. I'm not sure when it broke, though.